backend/backend/services/business_logic.py

```diff
@@ -13,7 +13,7 @@ async def get_test_result(context: Context, subject: str):
     summary_chain = context.chains.get_chains()['summary_chain']
```

Comment:
The range in the for loop should be range(2) instead of range(1) to ensure all iterations are executed.